package/MDAnalysis/coordinates/PDB.py

```diff
@@ -651,7 +656,9 @@ def _write_pdb_header(self):
             self.REMARK(*remarks)
         except AttributeError:
             pass
-        self.CRYST1(self.convert_dimensions_to_unitcell(u.trajectory.ts))
+
+        if u.dimensions is not None:
```

Comment:
Thought: Whether or not we follow the PDB standard here, do we want to maybe throw a user warning here if dimensions is `None`?

Reply:
I'm not sure about this, some warnings are more annoying than useful. My view here is that if there is no `CRYST1` record in the output it means that the information needed to write it was absent.

However I would be 100% for a warning if we want to set the dimensions automatically (to follow the PDB standard).